docs/ui/preferences/virtual-machine/emulation.md

```diff
@@ -7,23 +7,22 @@ title: Emulation (macOS)
   <link rel="canonical" href="https://docs.rancherdesktop.io/ui/preferences/virtual-machine/emulation"/>
 </head>
 
- ![](https://suse-rancher-media.s3.amazonaws.com/desktop/v1.10/preferences/macOS_virtualMachine_tabEmulation.png)
-
 ### QEMU
 
-The "[QEMU](https://www.qemu.org/documentation/)" option is enabled by default and is used to run a guest operating system.
+ ![](https://suse-rancher-media.s3.amazonaws.com/desktop/v1.10/preferences/macOS_virtualMachine_tabEmulation.png)
 
- ![](https://suse-rancher-media.s3.amazonaws.com/desktop/v1.10/preferences/macOS_virtualMachine_tabEmulation_vz.png)
+- The "[QEMU](https://www.qemu.org/documentation/)" option is enabled by default and is used to run a guest operating system.
 
 ### VZ
 
+ ![](https://suse-rancher-media.s3.amazonaws.com/desktop/v1.10/preferences/macOS_virtualMachine_tabEmulation_vz.png)
+
 :::caution warning
 
 This is an **experimental** setting. The setting is available for ARM machines on  macOS 13.3 or later, and Intel machines on macOS 13.0 or later.
 
 :::
 
-The "VZ" option can also be enabled to make use of the native macOS [Virtualization.Framework](https://developer.apple.com/documentation/virtualization) for running a guest machine.
-
-* VZ option:
+- The "VZ" option can also be enabled to make use of the native macOS [Virtualization.Framework](https://developer.apple.com/documentation/virtualization) for running a guest machine.
+  - **VZ option**:
```

Comment:
Nitpick: the bold should go around the text of the label (**VZ** option, not **VZ option**)